pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>com.microsoft.azure</groupId>
 	<artifactId>azure-relay</artifactId>
-	<version>0.0.1-PREVIEW</version>
+	<version>0.0.2-PREVIEW</version>
```

Comment:
>.2 [](start = 13, length = 2)

Are you just about to create a new release? That's typically when we increase the version--as a last step before publishing.